src/GeositeFramework/css/app-print.css

```diff
@@ -95,4 +119,16 @@
 
 .instructions {
     display: none;
-}
\ No newline at end of file
+}
+
+.legend-close {
+    visibility: hidden;
+}
+
+.dojoResizeHandle {
+    visibility: hidden !important;
+}
+
+#export-title {
```

Comment:
Wasn't sure where to put this comment, but according to issue #767, the print title should be center aligned.

![image](https://cloud.githubusercontent.com/assets/43062/21203194/66105344-c21f-11e6-9977-7d041b40ae00.png)